docs/jwst/rscd/description.rst

```diff
@@ -48,21 +48,57 @@ thus has no impact on the slopes determined for each integration.
 
 Algorithm
 _________
-The RSCD correction step applies the exponential decay correction based on coefficients in the  reset switch charge decay reference file. Based on READOUT pattern
-(FAST or SLOW) and  Subarray type (FULL or one of MIRI defined subarray types) the reference file contains
-the scale factor and decay time (tau)  for even and odd rows to corrected for the reset effects. The
-accumulated DN of the pixel  from the previous integration is estimated by extrapolating the ramp using the second to last 
-and third to last groups. For each pixel the group values are corrected according the formula:
+This correction is only applied to integrations > 1. 
+The RSCD correction step applies ab exponential decay correction based on coefficients in the  reset switch 
+charge decay reference file. The reference files are selected based on READOUT pattern
+(FAST or SLOW) and  Subarray type (FULL or one of MIRI defined subarray types).
+The reference file contains the information necessary to derive the the scale factor and decay time 
+to corrected for the reset effects. The correction depends if the pixel is from an even or odd row. 
 
-    corrected value = input vale + dn_accumulated * scale * exp(-T / tau),
+The correction to be added to the input data has the form::
 
-where T is the time since the last frame in the last integration.
+    corrected data = input data data + dn_accumulated * scale * exp(-T / tau)  (Equation 1) 
 
-The correction algorithm is slightly modified if the previous integration saturated. In this case the scale factor 
-in the above equation is calculated using an estimate of the what the last frame in the previous integration
-would have been if saturation did not exist. This estimated last frame is from a linear fit of the non-saturating
-groups in the ramp. 
- 
+
+where T is the time since the last group in the last integration, tau is the exponential time constant and 
```

Comment:
"time since the last group in the last integration"?? The exposure is done once the last group of the last integration has taken place, so how can we still be counting time elapsed after that point? Do you perhaps mean "time since the last group in the **previous** integration"?